src/ui/pages/HomePage/HomePage.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import { HomePage } from './HomePage';
+
+export default {
+  title: 'Pages/Home Page',
```

Comment:
```suggestion
  title: 'Pages/Home',
```

Jak masz tylko jedno story do komponentu to możesz je "wynieść w górę" żeby nie było duplikatu (tzn. porównaj jak na bocznym pasku pokazuje się teraz `HomePage` vs `LevelSelectPage`). Robisz to nadając eksportowi taką samą nazwę jak tytuł story. [Źródło do dalszego poczytania](https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy#single-story-hoisting).

Tutaj są dwie opcje, jedna tak jak pod powyższym linkiem z nazwanym importem, a drugą i prostszą podałem w sugestii. Home i tak wyświetla się w zakładce `Pages` także każdy wie o co chodzi.

Reply:
Właśnie nie wiedziałem jak to zrobić, aby nie było tego pod Home, więc dzięki za wskazówki :D